### PR TITLE
Add scenario to update a tag

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/test/java/cucumber/steps/SetComposedEndpointStep.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package cucumber.steps;
```
Comment (Raymundogs) [inline src/test/java/cucumber/steps/SetComposedEndpointStep.java:1]: You should add copyright

File: src/test/java/cucumber/steps/SetTagBodyStep.java [hunk 2]
```diff
@@ -26,9 +25,11 @@ public SetTagBodyStep(ApiRequestBuilder apiRequestBuilder, ApiResponse apiRespon
 
     @When("I set the tags body with following values:")
     public void setTagBody(final Map<String, String> body) {
-        List<String> trashList =  new ArrayList<>();
-        trashList.add(body.get("name"));
-        scenarioContext.setTrash("Tags Trash", trashList);
+        LinkedList<String> trashList =  new LinkedList<>();
+        if (body.get("name") != null) {
```
Comment (Raymundogs) [inline src/test/java/cucumber/steps/SetTagBodyStep.java:29]: is it ok that the name is hardcoded? It will always use that parameter?

Comment (JorgeRCV) [reply src/test/java/cucumber/steps/SetTagBodyStep.java:29]: Yes it is necessary, because Tags always uses name instead of an ID

**Review by Raymundogs — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

File: src/test/java/cucumber/steps/SetComposedEndpointStep.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Given("^I set the (.*) composed endpoint (.*)$")
+    public void setComposedEndpoint(final String featureName, final String endpoint) {
+        List<String> pathParamsList = getPathParamsFromEndpoint(endpoint);
+        apiRequestBuilder
+                .endpoint(endpoint)
+                .cleanParams();
+        apiRequestBuilder.pathParams(pathParamsList.get(0), scenarioContext.getEnvData(pathParamsList.get(0)))
+                .pathParams(pathParamsList.get(1), scenarioContext.getTrashList(featureName).getLast());
+
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/SetComposedEndpointStep.java:36]: this is not generic check the steps of ApiSteps

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**